tests/Authenticators/PrivateKeyAuthenticatorTest.php

```diff
@@ -33,7 +33,7 @@ public function testMakeSuccess()
 
     /**
      * @expectedException \InvalidArgumentException
-     * @expectedExceptionMessage You must provide a valid key file
+     * @expectedExceptionMessage You must inform a valid key file
```

Comment:
inform is the wrong word

Reply:
I know that inform is the wrong word but that's what the upstream provider is using. I actually changed it once to have tests fail. Hence I changed it back. 

I've already created an upstream PR.

https://github.com/lcobucci/jwt/pull/283

Reply:
Thanks. I've left a comment on your upstream PR.